ci/container_scripts/install_deps.sh

```diff
@@ -39,6 +39,8 @@ esac
 APT_CI_PACKAGES=(
   curl
   rsync
+  libcap-dev
+  iproute2
```

Comment:
Yes, they would. Is there anything I have to do more for this?

Reply:
Yes, I think these deps should be moved into the `APT_PACKAGES` and `RPM_PACKAGES` sections and removed from the `APT_CI_PACKAGES` and `RPM_CI_PACKAGES` sections. 

But actually, now that I looked again I am reminded that the tests you added are only enabled in the `EXTRA` configuration. I see that it's not clear how to add dependencies that only apply to EXTRA tests. @stevenengler do we have a way to add deps only for EXTRA tests? Or what is the preferred way to handle this?

Reply:
I would say do whatever we do for Go programs which also only run under the `--extra` configuration.

So since we list the `golang-go` package under `APT_PACKAGES`, I would put `libcap-dev` and `iproute2` there as well instead of under `APT_CI_PACKAGES` (and the same for the RPM packages below). The `APT_CI_PACKAGES` was originally meant for things used by the CI workflow yaml and bash scripts.

But I think that's all that needs to be done. Since they're "extra" tests, we don't need to list these dependencies anywhere else.